README.md

```diff
@@ -10,12 +10,10 @@ A TypeScript library that converts GraphQL query and mutation strings into struc
 
 ## Features
 
-- ✅ **Full GraphQL Support**: Queries, mutations, and subscriptions
```

Comment:
I don't think we support subscriptions

Reply:
Amazing on that PR! I have brought back the comment with commit 6d860e7553f62f98571e5a30e8379654e73b4727